docs/sample_generator.rst

```diff
@@ -55,6 +56,12 @@ input_type = modinput | scripted_input | syslog_tcp | file_monitor | windows_inp
     * The way with which the sample data is ingested in Splunk depends on Splunk. The most similar ingesting approach is used for each input_type to get accurate index-time testing.
     * For example, in an Add-on, a sourcetype "alert" is ingested through syslog in live environment, provide input_type=syslog_tcp.
 
+index = <index>
+    * The index used to ingest the data.
+    * The index must be configured beforehand.
+    * If the index is not available then the data will not get ingested into Splunk and a warning message will be printed.
+    * Custom input is not supported for syslog_tcp or syslog_udp
```

Comment:
Looks like a type. "Custom Input". I think it should be "Custom index"

Reply:
Thanks, this has been updated and corrected.